awx/ui_next/src/components/DataListToolbar/DataListToolbar.jsx

```diff
@@ -28,6 +33,9 @@ const ToolbarContent = styled(PFToolbarContent)`
 `;
 
 function DataListToolbar({
+  isAllExpanded,
+  showExpandAll = false,
+  onExpandAll,
```

Comment:
I wonder if we can reduce the number of props here a bit and get rid of `showExpandAll` by checking if `onExpandAll` is falsy (and same for `showSelectAll`/`onSelectAll`)